sdk/identity/identity/src/credentials/defaultAzureCredential.ts

```diff
@@ -35,6 +36,15 @@ export class DefaultAzureCredential extends ChainedTokenCredential {
     credentials.push(new AzureCliCredential());
```

Comment:
Shouldn't `tokenCredentialOptions` also be passed to the `AzureCliCredential`?

Reply:
Possibly.  It doesn't currently accept or use the settings. We can take on some work for the next preview to bring AzureCli to the same style that the other credentials are using.